docs/release-checklist.rst

```diff
@@ -95,8 +129,10 @@ they will need to evaluate which contributors' signatures they trust.
 - (all steps above are completed)
 - sign the release
 
-  - git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-1.15.0rc0" tahoe-lafs-1.15.0rc0
-  - (replace the key-id above with your own)
+  - ``git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-1.16.0rc0" tahoe-lafs-1.16.0rc0``
```

Comment:
ditto comment about not using the markup (to facilitate copy-pasting).

Reply:
I think that's the markup to specify it's a "command" or "code" that way github give's special highlighting which makes it easier to copy.

Reply:
I don't read it in github though. Do you? Does @exarkun?

Reply:
I don't read it on github either.  I might read it on readthedocs, I guess.  It seems weird to shy away from rst in a rst file though.  Should it just be a txt file instead?

Or, probably a better compromise would actually be to use a code block instead of inline markup.  That is:

```
::
    some shell
```

Or

```
.. code-block:: sh

    some shell
```

I don't remember exactly how you fit that into a bulleted list.  I guess either there is some way or there isn't.

Reply:
My guess is that, whatever is using this file to do some kind of presentation is doing so based on it's extension. For example I usually do read most of the documentation on github itself or inside vscode that has extensions to present `markdown`, `restructedText`, `html` any common presentation lang.

So not using the semantics of the language itself makes the file less consistent across the  different presentation tools.

Reply:
okay, sounds like mixed opinions, so I guess that should mean "use the semantic thing" .. or as jean-paul suggests, maybe it should just be a .txt file?

(As per my other comment, somewhere, this is a pretty narrowly-focused document intended to speed up making a release for those few people who do that .. and I found it easier to copy-pasta the commands without surrounding stuff .. maybe just a space between the opening / closing code markups could be a good compromise?)

Reply:
There's already plenty of markup in so many places, removing them to make this a text file does not sound like the way to go.

Since the audience is narrow as you rightly said I would add those spaces, I would have love to remove it complete BUT I am guessing one of the goals of this edit in the first place to expand the audience for this process. 

Also it won't mean a lot? Once there's a script to automate this?